radis/misc/utils.py

```diff
@@ -12,9 +12,11 @@
 
 import os
 import sys
-from os.path import dirname
+from os.path import dirname, basename
 import importlib
 import inspect
+import fnmatch
+from radis.test.utils import getTestFile
```

Comment:
i don't think you need this one here, and i actually believe it creates a circular reference that breaks things

Reply:
Understood and corrected, thanks!

Reply:
imports can be a tedious thing!